collection-scripts/common.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+export BASE_COLLECTION_PATH="${BASE_COLLECTION_PATH:-/must-gather}"
+declare -a DEFAULT_NAMESPACES=("openstack" "openstack-operators")
```

Comment:
I think we should also include `baremetal-operator-system` `openshift-machine-api` namespace as we need to collect the logs for baremetal-operator and ironic used for baremetal provisioning.

We added something similiar in zuul collect logs recently: https://github.com/openstack-k8s-operators/ci-framework/commit/c760642f480e5277f2ae6da832dcfed4a6c8220d

Reply:
+1 and Done